src/Plaid/Management/PlaidLinkResponse.cs

```diff
@@ -26,18 +26,40 @@ public class PlaidLinkResponse
 		/// </summary>
 		/// <value>The accounts.</value>
 		[JsonProperty("accounts")]
+		[System.Diagnostics.CodeAnalysis.SuppressMessage(
+			"Performance",
+			"CA1819:Properties should not return arrays",
+			Justification = "Matching the data structure from the Plaid API")]
 		public AccountInfo[] Accounts { get; set; } = null!;
 
+		/// <summary>
+		/// The account object is not null in the onSuccess callback when 
+		/// your users connect a specific bank account via the Select Account 
+		/// pane in Link. You can enable this behavior in the Plaid Dashboard 
+		/// or by integrating with all Auth features.
+		/// 
+		/// Note: If Select Account is enabled for multiple accounts, account 
+		/// will contain only the first account. All accounts selected by the 
+		/// user will be present in the accounts object.
+		/// </summary>
+		[JsonProperty("account")]
+		public AccountInfo Account { get; set; } = null!;
+
 		/// <summary>
 		/// Gets or sets the institution.
 		/// </summary>
 		/// <value>The institution.</value>
 		[JsonProperty("institution")]
 		public InstitutionInfo Institution { get; set; } = null!;
 
+
 		/// <summary>
 		/// Represents an <see cref="Entity.Account"/> metadata.
 		/// </summary>
+		[System.Diagnostics.CodeAnalysis.SuppressMessage(
```

Comment:
I would probably update the `.editorconfig` file to disable CA1034 rather than suppress it one by one. There are several specific codes already in there for examples.